packages/compass-indexes/src/components/indexes-table/property-field.tsx

```diff
@@ -78,6 +86,12 @@ const PropertyField: React.FunctionComponent<PropertyFieldProps> = ({
           link={getIndexHelpLink(cardinality.toUpperCase() as any) ?? '#'}
         />
       )}
+      {extra.status === 'inprogress' && (
+        <Badge variant={BadgeVariant.Blue}>In Progress...</Badge>
+      )}
+      {extra.status === 'failed' && (
+        <Badge variant={BadgeVariant.Red}>Failed</Badge>
```

Comment:
What about the failure reason? Should we add a tooltip with the error message or something?

Reply:
Pushed changes according to your comments 👍 so now for in progress modal we hide the "Create index" button and replace the "Cancel" name with "Close". Also replaced "Create in Progress" with "Index creation in progress. The dialog can be closed.".